src/Components/FeelingRoom/FeelingRoom.js

```diff
@@ -78,27 +92,44 @@ class FeelingRoom extends Component {
   startFeeling = () => {
     this.setState({isFeeling: true})
   }
+  playSound(e) {
+    let audio = e.target.parentNode.parentNode.firstChild
```

Comment:
I am deeply impressed by how you targeted this. I would love a walk through on how this exactly works next time we meet!